VMware.CloudFoundation.Reporting.psd1

```diff
@@ -12,7 +12,7 @@
     RootModule = '.\VMware.CloudFoundation.Reporting.psm1'
     
     # Version number of this module.
-    ModuleVersion = '2.1.0.1002'
+    ModuleVersion = '2.1.0.1003'
```

Comment:
Your merge details saying update 1003 > 1004 but here its 1002 > 1003 ?

Reply:
Resolved in 343e932d8ec8e2e0039d3a5ba19c568a5ef72754.

#131 needs to be reviewed and merged prior.